internal/pkg/statemanager/manager.go

```diff
@@ -106,3 +107,14 @@ func (m *manager) DeleteState(key StateKey) error {
 	m.statesMap.Delete(key)
 	return m.Write(m.GetStatesMap().Format())
 }
+
+func (m *manager) GetOutputs(key StateKey) (interface{}, error) {
+	state := m.GetState(key)
+	if state == nil {
+		return nil, fmt.Errorf("cannot find state by key: %s", key)
+	}
+	if value, ok := state.Resource["outputs"]; ok {
```

Comment:
if value, ok := state.Resource["outputs"]; !ok {
return nil, fmt.Errorf("cannot find outputs from state: %s", state.Name)
}
return value, nil

Reply:
value, ok can only be used in if{} structure,  so if like this format,  will add "var value interfase{}" define

Reply:
```go
value, ok := state.Resource["outputs"]
if !ok {
return nil, fmt.Errorf("cannot find outputs from state: %s", state.Name)
}
return value, nil
```

It seems easier to put the normal return value at the end